Sources/PointFree/Account/Account.swift

```diff
@@ -40,7 +40,7 @@ private func fetchAccountData<I>(
 
   let (user, subscriberState) = lower(conn.data)
 
-  let userSubscription = user.subscriptionId
+  let userSubscription: EitherIO<Error, Models.Subscription> = user.subscriptionId
```

Comment:
type inference probs?

Reply:
yeah same as with Swift 5 :( thinks it's an optional